README.md

```diff
@@ -94,15 +96,18 @@ export DRYAD_AWS_ACCESS_KEY_ID=<your access key ID, in single quotes>
 export DRYAD_AWS_SECRET_ACCESS_KEY=<your secret access key, in single quotes>
 export DRYAD_AWS_KEYPAIR_NAME=<the name of your keypair, in single quotes>
 export DRYAD_AWS_PRIVATEKEY_PATH=<the full path to your .pem.txt file, (e.g. ~/.ssh/user.pem.txt) in single quotes>
+export DRYAD_AWS_VM_NAME=<the name you want the VM to have in the EC2 console>
 ```
 
 Reload your settings when you're done: `source ~/.bash_profile`.
 
 Verify that you have the correct path specified: `more $DRYAD_AWS_PRIVATEKEY_PATH` should give you a cryptic key starting with `-----BEGIN RSA PRIVATE KEY-----`. If not, double-check your path in your .bash_profile and source it again.
 
+Finally, it is good practice to provide a descriptive name for the VM to distinguish it from other VMs on the AWS system. Edit the `Vagrantfile`, and change the name within the `aws.tags` section.
```

Comment:
It looks like you don't need to have this sentence, because you've set the Vagrantfile to pull the name from an env variable.

Reply:
When I launched my instance, the name did not get picked up by AWS. I had to edit the name manually at the management console.

Reply:
After more investigation, I see that the name is in the Tags.